episodes/chapter3.md

```diff
@@ -0,0 +1,120 @@
+
+Following vantage6's concepts, this scenario would involve two collaborations, one for each research project. As described in previous episodes, a vantage6 node is needed for each collaboration. Consequently, as illustrated in the diagram below, both CANTABRIA and LIFELINES organizations require two vantage6 data-node instances each. Since each data node defines its own rules for data access and algorithm usage, this ensures that analyses from various collaborations, even those involving shared organizations/datasets, will not conflict with one another.
+
+![Hypothetical collaborations scenario](fig/chapter3/orgs_n_collabs_scenario.png)
+
+
+## Algorithms trustworthiness on a federated setting
+
+While a technical infrastructure like the one previously described offers a strong defense against many data privacy risks, there remains one crucial security aspect that falls outside the platform's scope: the validation of the code that will run on this infrastructure. For instance, the administrators of the nodes running within each organization are responsible for defining which algorithms (i.e., [which the docker images](https://docs.vantage6.ai/en/main/node/configure.html#all-configuration-options)) will be allowed for execution on the respective collaborations. As this is a critical and complex task that entails activities like code analysis and verification, working with algorithms from trusted sources is the primary line of defense against potential threats.
+
+Vantage6's algorithm store feature aims to enhance trustworthiness by offering a centralized platform for managing pre-registered algorithms. This serves as an alternative to using algorithms from unknown authors or those lacking transparency regarding their development process and status. The algorithm store currently allows researchers to explore which algorithms are available and how to run them. This, along with its integration with vantage6's UI, streamlines task execution requests within collaborations.
+
+As of the time of writing this tutorial, efforts are underway to integrate additional information to the algorithms metadata such as creators and code reviewers. Moreover, plans are in place to incorporate the algorithm review process into the publication procedure for any algorithms in the store.
+
+## Running a PET (privacy-enhancing technology) analysis without programming!
+
+In this episode, you will perform a PET analysis on an existing vantage6 collaborations (based on 'dummy' nodes) that resemble the two described above. For reference, the datasets of each organization can be seen here (TODO).
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge 1: understanding a simple federated algorithm
```

Comment:
This challenge is important for this chapter, but after refining it as suggested in your previous comment: it should be focused not on understanding the concept of partial/central, but on understanding how the particular algorithm used in the episode works, as this is key for the follow-up questions. I think, for learning purposes, it would be better to use different algorithms in episodes 2 and 3. In EP. 2 a simple one for illustrative purposes, and in EP. 3 a more complex one (not 'average' :)) that requires a bit more analysis by the participants. Maybe we can discuss later which alternative ones could fit the proposed collaborations scenario. In the meantime, I'll address the remaining comments as this episode would still be based on the 'average' algorithm.